### PR TITLE
refactor: remove license headers from files

### PR DESCRIPTION
**Pull request by canvural**

### CODE REVIEW

**Comment by szepeviktor** [comment]
@nunomaduro New era of license-free source?

File: src/Properties/SchemaTable.php [hunk 1]
```diff
@@ -3,12 +3,12 @@
 declare(strict_types=1);
 
 /**
- * This file is part of Larastan.
+ * This file uses parts of code from following projects.
```
Comment (nunomaduro) [inline src/Properties/SchemaTable.php]: I would remove this too.

File: src/Properties/SchemaColumn.php [hunk 2]
```diff
@@ -3,12 +3,12 @@
 declare(strict_types=1);
 
 /**
- * This file is part of Larastan.
+ * This file uses parts of code from following projects.
```
Comment (nunomaduro) [inline src/Properties/SchemaColumn.php]: I would remove this too.

File: src/Properties/SchemaAggregator.php [hunk 3]
```diff
@@ -3,12 +3,12 @@
 declare(strict_types=1);
 
 /**
- * This file is part of Larastan.
+ * This file uses parts of code from following projects.
```
Comment (nunomaduro) [inline src/Properties/SchemaAggregator.php]: I would remove this too.

Comment (nunomaduro) [reply src/Properties/SchemaAggregator.php]: Maybe just add a note on the class phpdocs.

File: src/Properties/SchemaColumn.php (hunk 2 above)
Comment (nunomaduro) [reply src/Properties/SchemaColumn.php]: Maybe just add a note on the class phpdocs.

File: src/Properties/SchemaTable.php (hunk 1 above)
Comment (nunomaduro) [reply src/Properties/SchemaTable.php]: Maybe just add a note on the class phpdocs.

Comment (canvural) [reply src/Properties/SchemaTable.php]: How should that look like?

Comment (nunomaduro) [reply src/Properties/SchemaTable.php]:
```php
/**
 * This code was inspired by ..
 */
class Foo {}
```

File: src/Properties/SchemaAggregator.php (hunk 3 above)
Comment (szepeviktor) [reply src/Properties/SchemaAggregator.php]: @nunomaduro There is such a thing as PHPDoc file comment.

```php
/**
 * @package Larastan
 * @license GPL
 */
```

and so on https://docs.phpdoc.org/latest/references/phpdoc/index.html

Comment (nunomaduro) [reply src/Properties/SchemaAggregator.php]: Nah, I rather keep it simple with a link to the copy/pasted code.

Comment (canvural) [reply src/Properties/SchemaAggregator.php]: Updated.

File: src/Properties/SchemaColumn.php (hunk 2 above)
Comment (canvural) [reply src/Properties/SchemaColumn.php]: Updated.

File: src/Properties/SchemaTable.php (hunk 1 above)
Comment (canvural) [reply src/Properties/SchemaTable.php]: Updated.

**Comment by troelsselch** [comment]
@canvural The PR I did for `AppExtension` (https://github.com/nunomaduro/larastan/pull/431) includes a copyright notice. Perhaps that should also be removed? https://github.com/nunomaduro/larastan/blob/master/src/ReturnTypes/Helpers/AppExtension.php

**Comment by nunomaduro** [comment]
Yeah - should be removed.

**Pull request merged**